d3-cloud/resources/cljsjs/d3-cloud/common/d3-cloud.ext.js

```diff
@@ -0,0 +1,2 @@
+var d3 = {};
+d3.layout.cloud = function () {};
```

Comment:
it work's locally here
I don't really know how externs works, but d3 is in dependencies so maybe it makes it work